debuglogger/default.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package debuglogger
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type defaultEvent struct {
+	level  LogLevel
+	logger *log.Logger
+	fields []byte
+}
+
+func (e *defaultEvent) Msg(msg string) {
+	if len(msg) == 0 {
+		return
+	}
+
+	s := strings.Builder{}
```

Comment:
Think `e.logger.Printf` would be better than using this builder to avoid the intermediate string.

`e.logger.Printf("[%s] %s%s\n", e.level.String(), msg, e.fields)`